roles/boot_iso/tasks/pxe.yml

```
@@ -0,0 +1,119 @@
+---
+
+- name: prepare PXE
+  block:
+    - name: ensure the required packgaes installed
+      package:
+        name: "{{ item }}"
+        state: present
+      with_items:
+        - syslinux
+        - tftp-server
+        - dnsmasq
+        - ipmitool
+        - httpd
+        - python3-pip
+
+    - name: Install multi python packages with version specifiers
```

Maybe you can name it like
```suggestion
    - name: Install required python packages
```

---

this is the old file that I will delete in the future
the new file is
https://github.com/halleelyasaf/crucible/blob/add_pxe_boot/roles/setup_ipmi_tools/tasks/main.yml